kube-runtime/src/controller/mod.rs

```diff
@@ -325,11 +328,17 @@ where
 {
     /// Create a Controller on a type `K`
     ///
-    /// Configure `ListParams` and `Api` so you only get reconcile events
-    /// for the correct `Api` scope (cluster/all/namespaced), or `ListParams` subset
+    /// Takes an [`Api`] object that determines how the `Controller` listens for changes to the `K`.
+    ///
+    /// The [`ListParams`] lets you define a possible subset of objects of `K` that you want to control
+    /// and receive reconcile events for.
+    /// For the full set of objects `K`, you can use `ListParams::default`.
+    ///
+    /// This variant constructor is for [`dynamic`] types found through discovery. Prefer [`Controller::new`] for static types.
     ///
-    /// Unlike `new`, this function accepts `K::DynamicType` so it can be used with dynamic
-    /// resources.
+    /// [`ListParams`]: https://docs.rs/kube/*/kube/api/struct.ListParams.html
+    /// [`Api`]: https://docs.rs/kube/*/kube/api/struct.Api.html
+    /// [`dynamic`]: https://docs.rs/kube/*/kube/core/dynamic/index.html
```

Comment:
IIRC you can put the type path here instead (such as `kube::core::dynamic`)

Reply:
ah, cross-crate links didn't use to work before, but it looks like they do now, changing!